rtengine/dcraw.cc

```diff
@@ -9078,7 +9078,7 @@ void CLASS adobe_coeff (const char *make, const char *model)
     RT_matrix_from_constant = ThreeValBool::T;
   }
   // -- RT --------------------------------------------------------------------
-  
+
```

Comment:
Sorry about this... (auto-trimming whitespace in my editor...)

Reply:
No need to be sorry. This is a valid change. The one adding unnecessary whitespace made a mistake.